yt/go/yt/interface.go

```diff
@@ -37,6 +37,7 @@ import (
 	"io"
 
 	"go.ytsaurus.tech/library/go/core/xerrors"
+
```

Comment:
Usually we group import into 3 groups
```
import (
    stdlib // os, fmt
    
    external libs // github.com, gopkg.in
    
    local ligs // go.ytsaurus.tech
)
```
with no empty lines in each group

Reply:
Makes sense, updated goland's setting to group `go.ytsaurus.tech` and fixed.